packages/kitchen/src/pan/Text fields.vue

```diff
@@ -1094,7 +1109,7 @@
           loading
         >
           <v-progress-linear
-            slot="progress"
+            #progress
```

Comment:
This line is not using `<template>`. All other places are using it.

Reply:
this works like this :)

Reply:
It doesn't
https://codepen.io/anon/pen/XOoWMP